src/modules/newsletter-subscription/newsletter-subscripion.controller.ts

```diff
@@ -23,9 +24,26 @@ export class NewsletterSubscriptionController {
   @Get()
   @HttpCode(HttpStatus.OK)
   @ApiOperation({ summary: 'Fetch all subscribers to newsletter' })
-  @ApiResponse({ status: 200, type: [Object] })
-  findAll() {
-    return this.newsletterSubscriptionService.findAll();
+  @ApiResponse({
+    status: 200,
+    description: 'Return all team members',
+    schema: {
+      properties: {
+        status: { type: 'string' },
+        message: { type: 'string' },
+        data: {
+          type: 'array',
+          items: { $ref: '#/components/schemas/NewsletterSubscriptionResponseDto' },
+        },
+      },
+    },
+  })
+  async findAll(): Promise<{ message: string; data: NewsletterSubscriptionResponseDto[] }> {
+    const subscribers = await this.newsletterSubscriptionService.findAll();
+    return {
```

Comment:
Use a better descriptive name. 


```suggestion
  async findAllSubscribers(): Promise<{ message: string; data: NewsletterSubscriptionResponseDto[] }> {
    const subscribers = await this.newsletterSubscriptionService.findAllSubscribers();
    return {
```

Reply:
Returns should be done in the Service File